include/RAJA/util/basic_mempool.hpp

```diff
@@ -60,10 +60,17 @@ class MemoryArena
   using used_value_type = typename used_type::value_type;
 
   MemoryArena(void* ptr, size_t size)
-      : m_allocation{ptr, static_cast<char*>(ptr) + size},
-        m_free_space({free_value_type{ptr, static_cast<char*>(ptr) + size}}),
-        m_used_space()
+    : m_allocation{ ptr, static_cast<char*>(ptr)+size },
```

Comment:
Rather than splitting this, could we have the m_free_space take an explicit instantiation? I imagine the issue is probably the implicit conversion from the initializer list.

Reply:
done.